package.json

```diff
@@ -4,6 +4,7 @@
   "description": "CLI tool for syncing assets from and to Loco",
   "license": "MIT",
   "repository": "robrechtme/loco-cli",
+  "type": "module",
```

Comment:
Why is this needed?

Reply:
I've reverted it and fixed an issue I had in another, less intrusive way